lib/RxTouch.js

```diff
@@ -1,8 +1,105 @@
 const s3 = require("../lib/s3");
 
-module.exports.noAppointmentMatchString = "no locations with available appointments";
+async function jQueryPost(page, url, data) {
+    return await page.evaluate(
+        async (url, data) => {
+            return await new Promise((resolve) => {
+                $.post(url, data, function (data) {
+                    resolve(data);
+                });
+            });
+        },
+        url,
+        data
+    );
+}
 
-module.exports.ScrapeRxTouch = async function ScrapeRxTouch(browser, site, siteName) {
+async function GetAllAvailability(availabilityService, zip) {
+    const today = new Date();
```

Comment:
I'm trying to get us moved over to `moment` instead of Date() because it's so much easier to understand and format and everything. Would you mind making the switch?

Reply:
switched to moment